clients/edgex-foundry/util.go

```diff
@@ -0,0 +1,295 @@
+package edgex_foudry
```

Comment:
please add license header

Reply:
Thanks for reminding, the license header is added.